nuxeo-features/nuxeo-platform-imaging/nuxeo-platform-imaging-core/src/test/java/org/nuxeo/ecm/platform/picture/core/test/TestPictureConversions.java

```diff
@@ -234,6 +245,28 @@ public void shouldFilterPictureConversions() throws Exception {
         undeployContrib(PICTURE_CONVERSIONS_FILTERS_COMPONENT_LOCATION);
     }
 
+    @Test
+    public void pictureConversionsAlwaysHaveExtensions() throws IOException {
+        DocumentModel picture = session.createDocumentModel("/", "picture", "Picture");
+        // Use a small image so the biggest conversions will have the same result and it will be fetched from the cache
+        Blob blob = Blobs.createBlob(FileUtils.getResourceFileFromContext("images/cat.gif"));
+        blob.setFilename("Cat.gif");
+        blob.setMimeType("image/gif");
+        picture.setPropertyValue("file:content", (Serializable) blob);
+        picture = session.createDocument(picture);
+        txFeature.nextTransaction();
+
+        // Wait for the end of all the async works
+        eventService.waitForAsyncCompletion();
+
+        // Fetch the picture views
+        MultiviewPicture multiviewPicture = picture.getAdapter(MultiviewPicture.class);
+        for (PictureView pictureView : multiviewPicture.getViews()) {
```

Comment:
You should also assert on the number of views, otherwise if there are 0 then you're not testing anything.